phases/phase-1/challenges/posts/gaia

```diff
@@ -0,0 +1,2 @@
+B9FC92E2BD0695912F5C4B4174B554ED44F8150E672266C60FEC98A826FB0092
+E4B16D138C5A21BB7FCE53D8B14AAC9DDA61831BDD5A335D1B45897986262F7B
```

Comment:
I'm sorry, but this looks like a `Like` transaction. 

Please remove this file from the file and add it to the proper one by running: 

```
echo E4B16D138C5A21BB7FCE53D8B14AAC9DDA61831BDD5A335D1B45897986262F7B > ./phases/phase-1/challenges/likes/gaia
```